_docs/provider.yaml

```diff
@@ -2,5 +2,3 @@ host: http://localhost:3001
 attributes:
   - key: region
     value:  us-west
-  - key: tier
```

Comment:
it's for illustration

Reply:
That's true, I'll convert it back to be more of an example than an actual declaration.